src/styles/fields.styl

```diff
@@ -28,20 +28,23 @@
             margin-right 0
 
     .set-field
-        &:after
+        &:not(.set-field--checkbox):after,
+        &.set-field--checkbox .set-toggle label:after
```

Comment:
I am not convinced by all of this. 
If you have to cancel then reattribute a style so many time in the same file, maybe you should not put it in the first place. 
What i'm trying to say is : don't put the `set-field` class on this component. Duplicate this class with a new one and modify instead. 
Or you can also create some kind of HOC (High Order Class 🤣) `set-field` as a wrapper and then `set-field--input` for inputs specificity and `set-field--toggle` for toggles specificity.
You know what I'm saying?